docs/diagrams/ModelClassDiagram.puml

```diff
@@ -8,12 +8,12 @@ Package Model <<Rectangle>>{
 Class "<<interface>>\nReadOnlyAddressBook" as ReadOnlyAddressBook
 Class "<<interface>>\nReadOnlyUserPrefs" as ReadOnlyUserPrefs
 Class "<<interface>>\nModel" as Model
-Class AddressBook
+Class Scheduler
 Class ModelManager
 Class UserPrefs
 
-Class UniquePersonList
-Class Person
+Class uniqueEventList
```

Comment:
`uniqueEventList` should be a capital U, since its the name of a class.

Reply:
ok noted, to be fixed in v1.4